.github/workflows/deploy.yaml

```diff
@@ -92,9 +92,8 @@ jobs:
         if: ${{ !env.ACT }}
         run: ./scripts/deploy-smart-contracts.sh
         env:
-          DEPLOYER_WALLET_PRIVATE_KEY: ${{ secrets.DEPLOYER_WALLET_PRIVATE_KEY }}
+          DEPLOYER_PRIVATE_KEY: ${{ secrets.DEPLOYER_WALLET_PRIVATE_KEY }}
           ETHERSCAN_API_KEY: ${{ secrets.ETHERSCAN }}
-          BLOCKSCOUT_KEY: ${{ secrets.BLOCKSCOUT }}
```

Comment:
Did we drop smart-contract verification on Blockscout? Nevertheless, the env variable is nowhere used, so we can safely drop it.